HandyImage.user.js

```diff
@@ -2053,6 +2052,7 @@ function makeworld()
 	case "imagenimage.com":
 	case "imageshimage.com":
 	case "imagedax.net":
+	case "pornimagex.com"
```

Comment:
This website is connected to http://www.imagescash.com/.
and I wasn't able to skip the "continue to image". fix this please.

and Would you mind if you update how to skip timer or click "continue to image" on the tip?